TSOClient/tso.client/Utils/GameLocator/WindowsLocator.cs

```diff
@@ -15,6 +15,10 @@ public string FindTheSimsOnline()
         {
             string Software = "";
 
+            // Search relative directory similar to how macOS and Linux works; allows portability
+            string localDir = @"../The Sims Online\TSOClient\";
```

Comment:
`/` and then `\`? Then you replace it later anyways. For best cross platform support, we always use `/` since it also works on windows.